portfolio/src/main/webapp/index.html

```diff
@@ -54,7 +54,10 @@ <h1>Iris Liu</h1>
                             <p class="self-introduction">Welcome to my personal website. I hope you find this site helpful and inspiring. And feel free to contact me anytime</p>
                         </div>
                         <div>
-                            <button id="get-comment-button">Fetch a comment!</button>
+                            <form action="/data" method="POST">
+                                <textarea class="textarea" id="comment-input" name="comment-input" placeholder="Enter your comment..."></textarea>
+                                <input class="button" type="submit">
```

Comment:
The default text of the button is "Submit Query". It's fine of course but if you'd like to change it you can use the `value` attribute.